.github/workflows/labeler.yml

```
@@ -0,0 +1,16 @@
+name: Label pull request
+on:
+  pull_request_target:
+    branches:
+      - main
+    types:
+      - opened
+
```

This workflow only triggers on `pull_request_target` events of type `opened`, which means it won't re-evaluate labels if a pull request is updated (`synchronize`) or reopened (`reopened`).

Wondering this would be a better solution:

```suggestion
      - opened
      - synchronize
      - reopened
```

---

Reviewed before "request review". Add request review on osu-arti/termtable or other individual temmembers to satisfy review required branch protection rule.